tests/libraries/nested/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup "Prepare test"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        rlRun "set -o pipefail"
+        rlRun "tmt run -ar discover -vvvddd report -vvv 2>&1 >/dev/null | tee output"
```

Comment:
```suggestion
        rlRun -s "tmt run -ar discover -vvvddd report -vvv 2>&1 >/dev/null"
```

What about using `rlRun -s` to capture the output and dropping the `pipefail` above?